ci-operator/config/stolostron/must-gather/stolostron-must-gather-main.yaml

```diff
@@ -1,20 +1,16 @@
 base_images:
-  cli:
-    name: "4.7"
+  base:
```

Comment:
Could you explain what image tag these fields correspond to? I don't understand how this baseimage block translates into the image tag to pull

Reply:
Yes, I think that's correct, so it appears it works something like this (using the reference from the Dockerfile.prow):
```
registry.ci.openshift.org/ocp/4.7:cli
                          |   |   |
                          |   |   tag
                          |   name
                          namespace
```
And the to illustrate the benefit of using the `base_image` tags, when I didn't use them initially, the build took 10 minutes longer: https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/rehearse-27010-pull-ci-stolostron-must-gather-main-images

We should be able to find each of these images built somewhere in `release`, just like CICD builds our builder images in the `stolostron` namespace and we reference them in the `build_root` stanza.